src/Plugin/Action/EntityCreate.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\Action\EntityCreate.
+ */
+
+namespace Drupal\rules\Plugin\Action;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\rules\Core\RulesActionBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a generic 'Create a new entity' action.
+ *
+ * @Action(
+ *   id = "rules_entity_create",
+ *   deriver = "Drupal\rules\Plugin\Action\EntityCreateDeriver",
+ * )
+ *
+ * @todo: Add access callback information from Drupal 7.
+ */
+class EntityCreate extends RulesActionBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The entity storage service.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $storage;
+
+  /**
+   * The entity type id.
+   *
+   * @var string
+   */
+  protected $entityTypeId;
+
+  /**
+   * Constructs an EntityCreate object.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin ID for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Entity\EntityStorageInterface $storage
+   *   The entity storage service.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityStorageInterface $storage) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->storage = $storage;
+    $this->entityTypeId = $plugin_definition['entity_type_id'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity.manager')->getStorage($plugin_definition['entity_type_id'])
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function summary() {
+    return $this->t("Create a new $this->entityTypeId entity");
```

Comment:
This should use proper placeholders with ->t(). $this->entityTypeId should go into the parameter array and should not be concatenated directly into the translatable string.